app/views/lessons/show.html.erb

```diff
@@ -7,8 +7,29 @@
 
     <main class="grid grid-cols-12 gap-x-6" data-controller="lesson-toc" data-lesson-toc-item-classes-value="no-underline hover:text-gray-800 text-sm dark:hover:text-gray-300">
       <article class="col-span-full xl:col-span-7 xl:col-start-2">
-        <%= render ContentContainerComponent.new(classes: 'xl:mx-0', data_attributes: { lesson_toc_target: 'lessonContent' }) do %>
-          <%= @lesson.body.html_safe %>
+        <%= render ContentContainerComponent.new(classes: 'xl:mx-0', data_attributes: { lesson_toc_target: 'lessonContent' }) do |component| %>
+        <%= @lesson.body.html_safe %>
+
+        <% component.with_footer do %>
+          <div class="py-10  xl:col-span-7 xl:col-start-2 lg:col-start-3 lg:col-span-7">
+          <%= link_to github_edit_url(@lesson), target: :_blank, rel: 'noreferrer noopener', aria: {label: 'Improve this lesson on GitHub' }, class: 'hint--top underline hover:no-underline text-gray-600 hover:text-gray-800 dark:text-gray-400 dark:hover:text-gray-300 px-5' do %>
+ # Only display icon with tooltip at mobile width -%>
+          <span class="fab fa-github mr-1  "></span>
+ # Display text at larger viewport widths -%>
+          <span class=" ">
```

Comment:
Nit: Just a leftover empty class we can remove - we might be able to remove the span and just have the text on its own. But no harm in leaving it in either, totally up to you if you want to keep it.

```suggestion
          <span>
```